pkg/ui/installsupport.go

```diff
@@ -32,8 +32,8 @@ func InstallSupport(mux MuxInterface) {
 	prefix := "/static/"
 	mux.Handle(prefix, http.StripPrefix(prefix, fileServer))
 
-	// Expose files in third_party/swagger-ui/ on <host>/swagger-ui/
+	// Expose files in third_party/swagger-ui/ on <host>/
 	fileServer = http.FileServer(&assetfs.AssetFS{Asset: Asset, AssetDir: AssetDir, Prefix: "third_party/swagger-ui"})
-	prefix = "/swagger-ui/"
+	prefix = "/"
```

Comment:
I hadnt realized that swagger support was optional :)
Changed this now.
swagger-ui is exposed on /swagger-ui/ now. And it is exposed only if config.EnableSwaggerSupport is true.